js/main.js

```diff
@@ -0,0 +1,91 @@
+var button = document.getElementById('buttonNews');
+button.onclick = function displayAndHide() {
+    var hide = document.getElementById('displayInfo');
```

Comment:
**hide** is a bad variable name here - call it what it is, **infoPanel** or something like that